src/components/dashboard/Claim.js

```diff
@@ -214,29 +189,81 @@ const Claim = props => {
         <Section.Stack style={styles.extraInfo}>
           <Image source={illustration} style={styles.illustration} resizeMode="contain" />
           <Section.Row style={styles.extraInfoStats}>
-            <Section.Text fontWeight="bold">{numeral(claimedToday.people).format('0a')} </Section.Text>
-            <Section.Text>People Claimed </Section.Text>
-            <BigGoodDollar
-              number={claimedToday.amount}
-              formatter={number => numeral(number).format('0a')}
-              bigNumberProps={{ fontSize: 16 }}
-              bigNumberUnitProps={{ fontSize: 10 }}
-            />
-            <Section.Text>Today!</Section.Text>
+            <Section.Text fontWeight="bold">{numeral(state.claimedToday.people).format('0a')} </Section.Text>
+            <Section.Text>people have already claimed today!</Section.Text>
           </Section.Row>
-          <Section.Stack style={styles.extraInfoCountdown}>
-            <Section.Text style={styles.extraInfoCountdownTitle}>Next Daily Income:</Section.Text>
-            <Section.Text color="surface" fontFamily="slab" fontSize={36} fontWeight="bold">
-              {nextClaim}
-            </Section.Text>
-          </Section.Stack>
-          {ClaimButton}
+          {!isCitizen && <Countdown styles={styles} nextClaim={state.nextClaim} />}
+          <ClaimButton
+            isCitizen={isCitizen}
+            entitlement={state.entitlement}
+            nextClaim={state.nextClaim}
+            loading={loading}
+            onPress={() => (isCitizen && entitlement ? handleClaim() : !isCitizen && faceRecognition())}
```

Comment:
@fernandomg I am trying to run this branch on my local machine and found that there's an issue while not whitelisted user is clicking on claim button - nothing happens. Also, the back arrow button does not redirect back.

Reply:
@fernandomg this is we removed FR for now. it still pushes the route so you have to click multiple times back.
@yaroslav-fedyshyn-nordwhale we modified the code so once you register your are immediatly whitelisted so the case when a user isn't whitelisted shouldn't happen for now.

Reply:
oh, I see

Reply:
@sirpy, so I must remove the route push, and try to claim anyway? or just disable the functionality?